install/kubernetes/templates/istio-pilot.yaml.tmpl

```diff
@@ -8,12 +8,37 @@ data:
   mesh: |-
     # Uncomment the following line to enable mutual TLS between proxies
     # authPolicy: MUTUAL_TLS
+
+    ## Set the following variable to true to disable policy checks by the Mixer.
+    ## Note that metrics will still be reported to the Mixer.
+    disablePolicyChecks: false
+
+    # To disable the mixer completely (including metrics), comment out
+    # the following line
     mixerAddress: istio-mixer.{ISTIO_NAMESPACE}:9091
+
     discoveryAddress: istio-pilot.{ISTIO_NAMESPACE}:8080
+    # This setting determines how frequently should Envoy fetch and update
+    # its internal configuration from Istio Pilot. Lower refresh delay
+    # results in higher CPU utilization and potential performance loss in
+    # exchange for faster convergence. Tweak this value according to your setup.
+    discoveryRefreshDelay: 30s
+
+    # TCP connection timeout between Envoy & the application, and between Envoys.
+    connectTimeout: 10s
+
     ingressService: istio-ingress.{ISTIO_NAMESPACE}
     egressProxyAddress: istio-egress.{ISTIO_NAMESPACE}:80
     statsdUdpAddress: istio-mixer.{ISTIO_NAMESPACE}:9125
+
+    ## Set enableTracing to false to disable request tracing.
+    enableTracing: true
     zipkinAddress: zipkin.{ISTIO_NAMESPACE}:9411
```

Comment:
These settings won't take effect. Please update to the new API definitions. The parser will ignore any undefined fields.

Reply:
?
the SHA in pilot's WORKSPACE is at Sep 10th.
```    """,
    commit = "6838a2bfc42260cf0513552a64a457f1cf39041c",  # Sep 10, 2017
    remote = "https://github.com/istio/api.git",
)```